types/apostrophe/index.d.ts

```diff
@@ -0,0 +1,302 @@
+// Type definitions for apostrophe 2.67
+// Project: https://github.com/apostrophecms/apostrophe#readme
+// Definitions by: My Self <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
```

Comment:
The lower the TS version number is, the easier the typings are to consume.  At a glance, nothing in these definitions jumped out as requiring TS 2.8 - did I miss something?